apps/charterafrica/src/pages/api/v1/github/[path].page.js

```diff
@@ -0,0 +1,24 @@
+import processData from "@/charterafrica/lib/tools/processData";
+
+const sourceMap = {
+  processData,
```

Comment:
```suggestion
  sync-ecosystem,
```

Or something along those lines.